Core/clim-basic/pointer-tracking.lisp

```diff
@@ -172,128 +172,112 @@
 
 
 ;;; DRAG-OUTPUT-RECORD and DRAGGING-OUTPUT.
+(defun make-default-feedback-function (stream output-record erase repaint dx dy)
+  (declare (ignore stream output-record))
+  (lambda (record sheet x0 y0 x y action)
```

Comment:
drag-output-record may be invoked only on a stream. but when multiple-window is T it is expected to behave as for tracking-pointer, that is:

```
When the boolean multiple-windows is true, then the pointer will be tracked across multiple windows, otherwise is will be tracked only in the window corresponding to sheet.
```

in default-feedback-function we do take steps to prevent errors when we draw feedback on a sheet which is not output-recording-stream. When someone calls drag-output-record on a stream with multiple-window = nil, then there is no danger, and if they call it with multiple-window = t they better know what they are doing.

Reply:
I see.